gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java

```diff
@@ -13,46 +13,33 @@
  */
 package io.trino.gateway.ha.security;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.trino.gateway.ha.security.LbOAuthManager.OidcTokens;
+import org.intellij.lang.annotations.Language;
 import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
 
 public class TestOidcToken
 {
-    private static final Logger log = LoggerFactory.getLogger(TestOidcToken.class);
-
     @Test
     public void testParseTokenParamsGracefully()
+            throws Exception
     {
         // This test is to make sure that we simulate the condition where
         // the OIDC providers send additional parameters in the token
         // 'to_be_ignored' parameter should not cause any parsing exception
         // All the other parameters should have the correct values
-        OidcTokens oidcTokens = null;
-        try {
-            ObjectMapper objectMapper = new ObjectMapper();
-
-            String jsonStr = "{\"id_token\" : \"ABC235234\", "
-                    + "\"access_token\" : \"AcessABCD123\", "
-                    + "\"refresh_token\" : \"RefreshTKN\", "
-                    + "\"token_type\" : \"TOKENType\", "
-                    + "\"expires_in\" : \"123456\", "
-                    + "\"to_be_ignored\" : \"XYX123456\", "
-                    + "\"scope\" : \"global\" "
-                    + "}";
-
-            oidcTokens = objectMapper.readValue(jsonStr, OidcTokens.class);
-        }
-        catch (JsonProcessingException ex) {
-            log.error(ex.getMessage());
-            fail();
-        }
+        @Language("JSON")
+        String jsonStr = "{\"id_token\" : \"ABC235234\", "
```

Comment:
We should clean this up to a multi line string

Reply:
I didn't use it intentionally as it inserts new lines - the value will be different from the original text.

Reply:
Fair enough